scaling_performance/using_cluster_loader.adoc

```diff
@@ -111,12 +111,12 @@ identical namespaces will be appended to `Basename` to prevent collisions.
 which you deploy inside this namespace.
 
 |`configmaps`
-|A list of key-value pairs. Key is ConfigMap name, value is a path to a file to create
-the ConfigMap from.
+|A list of key-value pairs. The key is the `ConfigMap` name and the value is a
+path to a file from which you create the `ConfigMap`.
 
 |`secrets`
-|A list of key-value pairs. Key is Secret name, value is a path to a file to create
-the Secret from.
+|A list of key-value pairs. The key is the secret name and the value is a path to
```

Comment:
Inconsistency.  "Secret" and "ConfigMap" are OpenShift objects.  Why have one definition quoted and one unquoted?  Not sure what the best approach is, OpenShift documentation has this inconsistency too.  Understand you could be talking about a "secret" in a more general sense, though.